src/components/UserDetailsTooltip/index.js

```diff
@@ -28,11 +47,13 @@ function UserDetailsTooltip(props) {
                 </Text>
 
                 <Text style={[styles.textMicro, styles.fontColorReactionLabel]}>
-                    {String(userDetails.login || '').trim() && !_.isEqual(userDetails.login, userDetails.displayName) ? Str.removeSMSDomain(userDetails.login) : ''}
+                    {String(userDetails.login || '').trim() && !_.isEqual(formatLoginIdAsDisplayName(String(userDetails.login || '')), userDetails.displayName)
```

Comment:
May be this should be enough I guess
```suggestion
                    {String(userDetails.login || '').trim() && !_.isEqual(LocalePhoneNumber.formatPhoneNumber(userDetails.login || '')), userDetails.displayName)
```

Reply:
LocalePhoneNumber.formatPhoneNumber does all checks so we should be good to use that straightaway!